### PR TITLE
feat(packages/sui-theme): Add default values for thumbnail basic

### PR DESCRIPTION
**Pull request by izeller**
## Description
We need to add new default values for thumbnail basic.
Related PR -> https://github.com/SUI-Components/adevinta-spain-components/pull/435

## Related Issue
https://github.com/SUI-Components/adevinta-spain-components/pull/435

### CODE REVIEW

**Review by tonyghiani — APPROVED** [review]

**Review by rmoralp — APPROVED** [review]

File: packages/sui-theme/src/components/thumbnail/_basic.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$jc-thumbnail-basic-image: flex-start;
+$ar-thumbnail-basic-image: auto;
```
Comment (jcger) [inline packages/sui-theme/src/components/thumbnail/_basic.scss:2]: don't you need to set !default at the end so it can be overwritten?

**Review by nguasch — APPROVED** [review]

**Review by jcger — APPROVED** [review]

**Review by andresin87 — APPROVED** [review]

File: packages/sui-theme/src/components/thumbnail/_basic.scss [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$jc-thumbnail-basic-image: flex-start !default;
+$ar-thumbnail-basic-image: auto !default;
```
Comment (andresin87) [inline packages/sui-theme/src/components/thumbnail/_basic.scss:2]: what does `ar-` mean?

Comment (andresin87) [reply packages/sui-theme/src/components/thumbnail/_basic.scss:2]: aspect-ratio

Comment (nguasch) [reply packages/sui-theme/src/components/thumbnail/_basic.scss:2]: `aspect-ratio` I didn't find the abbreviation 😓

**Pull request merged**